earthfile2llb/local_state_cache.go

```diff
@@ -72,7 +72,9 @@ func createIncludePatterns(incl []string) []string {
 			inc = "./*"
 		} else if strings.HasSuffix(inc, "/.") {
 			inc = inc[:len(inc)-1] + "*"
+		} else {
```

Comment:
Why was this empty else added?

Reply:
leftover debug that wasn't fully removed, good catch.